README.md

```diff
@@ -56,43 +56,86 @@ Building:
 ---------
 Note that YOU are responsible to know how to use compilers and C++, the hints below are all the help I will give you:
 
-All Platforms:
-
-- TreeSheets requires the latest development wxWidgets from their repo:
+<details>

```

Comment:
why this html style stuff? lets stick to pure markdown, so the file is readable as a text file.